.travis.yml

```diff
@@ -16,9 +16,13 @@ python:
   - 3.4
   - nightly
 
+sudo: required # needed for trusty beta
```

Comment:
I don’t think this is needed anymore, see https://docs.travis-ci.com/user/trusty-ci-environment/.